pom.xml

```diff
@@ -52,25 +51,64 @@
     </dependencies>
 
     <build>
-        <pluginManagement>
-            <plugins>
-                <plugin>
-                    <groupId>net.wasdev.wlp.maven.plugins</groupId>
-                    <artifactId>liberty-maven-plugin</artifactId>
-                    <version>1.2.1</version>
-                </plugin>
-            </plugins>
-        </pluginManagement>
         <plugins>
             <plugin>
                 <groupId>net.wasdev.wlp.maven.plugins</groupId>
                 <artifactId>liberty-maven-plugin</artifactId>
+                <version>2.0-SNAPSHOT</version>
                 <configuration>
                     <configFile>src/main/wlp/server.xml</configFile>
                     <bootstrapProperties>
                         <appLocation>${project.build.finalName}</appLocation>
                     </bootstrapProperties>
+                    <looseApplication>true</looseApplication>
+                    <installAppPackages>project</installAppPackages>
                 </configuration>
+                <executions>
+	                <execution>
+	                    <id>stop-server-before-clean</id>
+	                    <phase>pre-clean</phase>
+	                    <goals>
+	                        <goal>stop-server</goal>
+	                    </goals>
+	                </execution>
+	                <execution>
+	                    <id>create-server</id>
+	                    <phase>prepare-package</phase>
+	                    <goals>
+	                        <goal>install-server</goal>
+	                        <goal>create-server</goal>
+	                    </goals>
+	                </execution>
+	                <execution>
+	                    <id>install-apps</id>
+	                    <phase>package</phase>
+	                    <goals>
+	                        <goal>install-apps</goal>
+	                    </goals>
+	                </execution>
+	                <execution>
+	                    <id>package-server</id>
+	                    <phase>package</phase>
+	                    <goals>
+	                        <goal>package-server</goal>
+	                    </goals>
+	                </execution>
+	                <execution>
+	                    <id>test-start-server</id>
```

Comment:
Since this project does not have any IT tests (i.e. no tests and no failsafe plugin), then you don't need the test-start-server or test-stop-server goals.